src/controllers/Controller.js

```diff
@@ -300,4 +304,78 @@ async loginRestaurant(req, res) {
     }
   }
 
+  async servePaymentForm(req, res) {
+    try {
+      const filePath = './public/payment-form.html';
+      fs.readFile(filePath, (err, data) => {
+        if (err) {
+          res.writeHead(500, { 'Content-Type': 'text/plain' });
+          res.end('Failed to fetch payment-form.html ' + ' ' + err);
```

Comment:
Can you please use the response method like down below
response(res, { status: 400, data: error.message });